src/main/java/Duke.java

```diff
@@ -1,10 +1,113 @@
+import main.java.*;
+import java.lang.StringBuilder;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> dukeList = new ArrayList<>();
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Greetings, I am\n" + logo);
+        System.out.println("How may I be of assistance today?");
+
+        String command = sc.nextLine();
+        while (!command.equals("bye")) {
+            if (command.equals("list")) {
+                System.out.println("These items are in your list: \n");
+                for (int i = 0; i < dukeList.size(); i++){
```

Comment:
Can consider using an UI object to print things out. i.e. abstract away the printing functions